src-ui/src/app/components/document-list/filter-editor/filter-editor.component.html

```diff
@@ -40,6 +40,13 @@
           (open)="onDocumentTypeDropdownOpen()"
           (selectionModelChange)="updateRules()"
           [allowSelectNone]="true"></app-filterable-dropdown>
+        <app-filterable-dropdown class="me-2 flex-fill" title="Storage path" icon="folder-fill" i18n-title
+          filterPlaceholder="Filter storage path" i18n-filterPlaceholder
+          [items]="storagePathes"
+          [(selectionModel)]="storagePathSelectionModel"
+          (open)="onStoragePathDropdownOpen()"
+          (selectionModelChange)="updateRules()"
+          [allowSelectNone]="true"></app-filterable-dropdown>
```

Comment:
```suggestion
```

Im not sure if this should be included in the filter editor? Not sure how others feel but the filter editor is already quite full and I definitely see the importance in the bulk editor (I want to change the storage path of multiple docs easily) but I dont see as much use for it here.

And if it is retained the HTML shoudl be adjusted to move the created/added buttons to a new line probably wrapping the 4 buttons in their own flex div and the added/created in another

Reply:
To clarify the second part I mean so it wraps better on mobile if this button is retained.

Reply:
If we hide it, it would still be available through the advanced search via `path:`. Hiding is an option, but probably not the best one.

What about use an icon for the dates, too? It should fit (at least for now).

Reply:
What about
- Calendar with arrow left/right 
- Clock with circular arrow around clock-wise, ccw

Reply:
Not sure where you got those, these are ones we use: https://icons.getbootstrap.com

Reply:
Forget my comment. You wrote before / after, but you meant created / added. And that’s not obvious with icons.

Reply:
Oh yea sorry thats what I meant yes.

Reply:
Now that I have write access I can do this.